src/theme/index.ts

```diff
@@ -60,6 +60,9 @@ export const theme = {
       },
     ],
   },
+  boxShadow: {
+    scrollToTop: '0px 4px 15px 0px #0D1017',
```

Comment:
Given this box shadow is specific to the scroll to top button, I wouldn't add it to the theme since it won't be reused. I'd just add the box shadow to the button instance through classes.